src/main/java/com/demonwav/mcdev/insight/ListenerEventAnnotator.java

```diff
@@ -68,6 +60,34 @@ public void annotate(@NotNull PsiElement element, @NotNull AnnotationHolder hold
             return;
         }
 
+        PsiCodeBlock methodCodeBlock = method.getBody();
+        if (methodCodeBlock == null) {
+            return;
+        }
+        PsiStatement[] methodBodyStatements = methodCodeBlock.getStatements();
+        for (PsiStatement statement : methodBodyStatements) {
+
+            PsiAnnotation eventHandler = null;
+
+            for (PsiAnnotation psiAnnotation : method.getModifierList().getAnnotations()) {
+                if (psiAnnotation.getQualifiedName().contains("EventHandler")) {
```

Comment:
Just like how the annotator already checks for whether the method description itself is valid, based on what modules are available.